ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity.component.ts

```diff
@@ -472,4 +476,25 @@ export class ActivityComponent extends BaseActivityComponent implements OnInit,
             });
         }
     }
+
+    private saveLastVisitedSectionIndex(sectionIndex: number): void {
+      if (this.shouldSaveLastStep && sectionIndex > this.model.sectionIndex) {
```

Comment:
It would be nice to have the same indent for new code lines like the whole file has.